imports/plugins/included/taxes-avalara/client/accounts/exemption.js

```diff
@@ -6,11 +6,16 @@ import { Packages, Accounts } from "/lib/collections";
 import { Accounts as AccountsSchema } from "/lib/collections/schemas/accounts";
 import { TaxEntityCodes } from "/client/collections";
 
+let entityCodeList = [];
+let currentAccount;
+
 Template.taxSettingsPanel.helpers({
   account() {
     const sub = Meteor.subscribe("UserAccount", this.member.userId);
```

Comment:
We created a separate publication so we can have a publication that takes in a userId and just returns the account with that id. Which is different from the way the existing [`Accounts` publication](https://github.com/reactioncommerce/reaction/blob/master/server/publications/collections/accounts.js#L9-L45) works.